README.md

```diff
@@ -1,2 +1,71 @@
 # docker-itchysats
-ItchySats in a Docker container
+
+A repo facilitating running ItchySats inside a [Docker][https://www.docker.com/what-docker] container on a x86-64 (e.g. macOS, Linux, Windows) or ARMv7 system.
+Docker website provides extensive tutorials on Docker installation, please refer there if you are new to Docker.
+
+The following guide assumes that you have Docker running, having [docker-compose][https://docs.docker.com/compose/install/] installed is also recommended.
+
+# Quickstart
+
+## Testnet
+
+If you just want to check out the application, and want to try out a "demo" version, where you don't trade with real money, then this config is for you.
+
+```
+docker-compose project-directory=testnet up -d
+```
+
+## Mainnet
+
+If you're ready to trade with real money, use the following command:
+
+```
+docker-compose project-directory=testnet up -d
```

Comment:
Can't we default to `mainnet` so that we don't need this for the `mainnet` setup? Would be more convenient.

Reply:
again, I opted for ease of onboarding and trying it out. If someone copy-pastes the code, it's just as easy (and if they want to be serious, they'll just copy the docker-compose.yml file).

Reply:
also, if anyone goes inside the mainet/testnet subdir, they can just call `docker-compose up` (I fully suspect that anyone who knows a bit of docker go this way IMHO). The guide is just to kickstart people who are new to it...